wear/src/main/res/layout/activity_contact_events.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  tools:context="com.alexstyl.specialdates.ContactEventsActivity"
+  tools:deviceIds="wear_round">
+
+  <LinearLayout
+    android:id="@+id/event_container"
```

Comment:
The project's naming convention for id names are: `screen_purpose_viewtype`.
This one for example could be `contact_events_event_container`